hashintel/hash
# H-1665, H-1778, H-1932: Improve plugin logs and default types, prevent auto-inference on app.hash.ai

## 🌟 What is the purpose of this PR?

A few smaller improvements to the browser plugin:
1. Show metadata and counting timer on pending requests (see demo)
2. Show copyable request id on request metadata (see demo)
    - this is the Temporal workflow id, which makes the request identifiable in Temporal and AWS logs. This isn't actually persisted anywhere in the graph – we need further discussions on storing provenance
    - this uses the label to indicate status rather than a tooltip as MUI `Tooltip` doesn't work in the browser plugin (I haven't really looked into this much, maybe there's an easy fix)
3. Show a gray dash instead of a green tick if a request completed without error but did not result in any entities being created or updated
4. Update the Crunchbase types
5. Prevent auto-inference running on app.hash.ai (stops rules configured for 'any domain' running when someone visits it to inspect entities inferred – they can trigger one-off inferences if this is desired for some reason)

This PR is based on #3900 because it depends on `DashIcon` which was made available to the plugin there (to show against unchanged entities).

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:

- [x] modifies the plugin which I will publish after this is merged

### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## 🛡 What tests cover this?



- Mostly manual, although the browser plugin has basic coverage of config now.

## ❓ How to test this?



1. Run an inference to see the behavior in the demo
2. Check autoinference rules for 'any' domain don't run on app.hash.ai

## 📹 Demo

https://github.com/hashintel/hash/assets/37743469/60cac325-a218-46bc-af86-5e33d485d307
